Documentation: Update README cli options (#1548)
iotaledger/iri: README.md

### Command Line Options 

For a full list of command line options that is up to par with the latest release see https://docs.iota.org/docs/node-software/0.1/iri/references/iri-configuration-options/.

> You can also run IRI with the `-h` or `--help` flag to print a description of all the configurations that are up to par with the latest commit.

Reviewer: ```suggestion
You can also run IRI with the `-h` or `--help` flag to print a description of all the configurations that are available in your version of IRI.
```
